include/seastar/core/smp.hh

```diff
@@ -101,13 +101,19 @@ struct smp_service_group_config {
 /// may form a cycle (and risking an ABBA deadlock). Create a
 /// new smp_service_group_instead.
 class smp_service_group {
+#ifndef SEASTAR_DEBUG
     unsigned _id;
+#else
+    uint16_t _id;
```

Comment:
This gratuitous change is asking for trouble.

Reply:
Saving space here isn't important, especially in debug mode. Avoiding unnecessary changes is.